src/dft/src/stitch/ScanStitch.cpp

```diff
@@ -35,6 +36,7 @@ void ScanStitch::Stitch(
   for (const std::unique_ptr<ScanChain>& scan_chain : scan_chains) {
     Stitch(block, *scan_chain, scan_enable);
   }
+  sta_->networkChanged();
```

Comment:
sta should be getting callbacks from odb dircectly.  This suggests some issue with callbacks that we should figure out rather than have to manually call this (which is also less efficient).

Reply:
I think this will take a while to debug, so for now I will cancel this PR. I think I found in dbInst where the calls to the callbacks are being made so I will try to add some logs there.

Thanks!,
Felipe